core/src/main/java/hudson/Platform.java

```diff
@@ -47,7 +47,7 @@
      */
     public final char pathSeparator;
 
-    private Platform(char pathSeparator) {
+    Platform(char pathSeparator) {
```

Comment:
why?

Reply:
https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html
Note: The constructor for an enum type must be package-private or private access. It automatically creates the constants that are defined at the beginning of the enum body. You cannot invoke an enum constructor yourself.

Reply:
Redundant keyword, just like `public` inside an interface, or static inner classes of an interface, or static inner enums (all redundant modifiers).